recipe/meta.yaml

```
@@ -69,14 +65,14 @@ outputs:
       build:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
-        - bazel <=3.4.1  # [not win]
-        - bazel <3.7     # [win]
+        - bazel =4.2.1
         - sysroot_linux-64 2.17  # [linux64]
         - curl
-        - cython >=0.29
+        - cython >=0.29.15
         - make
+        - python <3.10
       host:
-        - python
+        - python <3.10
```

Revert this change (here and everywhere else). 
```suggestion
        - python
```